cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_detail.spec.js

```diff
@@ -23,26 +41,29 @@ if (Cypress.env('WORKSPACE_ENABLED')) {
           permissions: {
             library_write: { users: ['%me%'] },
             write: { users: ['%me%'] },
+            library_read: { users: [NONE_DASHBOARDS_ADMIN_USERNAME] },
+            read: { users: [NONE_DASHBOARDS_ADMIN_USERNAME] },
           },
         },
       }).then((value) => (workspaceId = value));
     });
 
-    beforeEach(() => {
-      // Visit workspace update page
-      miscUtils.visitPage(`w/${workspaceId}/app/workspace_detail`);
-
-      cy.intercept('PUT', `/w/${workspaceId}/api/workspaces/${workspaceId}`).as(
-        'updateWorkspaceRequest'
-      );
-    });
-
     after(() => {
       cy.deleteWorkspaceById(workspaceId);
+      if (Cypress.env('SECURITY_ENABLED')) {
+        cy.deleteInternalUser(NONE_DASHBOARDS_ADMIN_USERNAME);
```

Comment:
should we delete role and role mapping as well?

Reply:
It won't fail when creating a role or mapping with the same name. Deleting a user is to protect the system. Will add the remove role logic.